packages/material-ui-codemod/README.md

```diff
@@ -465,7 +465,7 @@ The diff should look like this:
 ```
 
 ```sh
-npx jscodeshift --extensions js,ts,jsx,tsx --parser tsx -t node_modules/@material-ui/codemod/v0.15.0/import-path.js ./src
+npx jscodeshift --extensions js,ts,jsx,tsx --parser tsx -t node_modules/@material-ui/codemod/v0.15.0/import-path.js
```

Comment:
What do you think about adding `./src` back? I believe it was included to guide developers into how to target a folder.

Reply:
Or `<path>` ?